alien4cloud-core/src/main/java/alien4cloud/deployment/DeploymentTopologyService.java

```diff
@@ -562,11 +580,9 @@ private void checkGroups(Map<String, String> groupsLocationsMapping) {
      * @return
      */
     public Map<String, PropertyValue> processForDeployment(DeploymentTopology deploymentTopology, ApplicationEnvironment environment) {
```

Comment:
> Previous code was actually changing the deployment topology through injectInputValues meaning that the deployment topology had no more get_input on properties but only values

This is done now when setting the location or updating an input value.
So when calling this method, the inputs that have values have already been injected in the deployment topology